integ/src/main.rs

```diff
@@ -16,6 +20,8 @@ mod embedded {
     embed_migrations!("../arroyo-api/migrations");
 }
 
+include!("../../arroyo-openapi/client/src/lib.rs");
```

Comment:
The client is included like this since it's generated by arroyo-openapi's build.rs. I'm not sure if there's a better way.

Reply:
I think you can depend on the arroyo-openapi crate in the integ Cargo.toml instead of including the file here

Reply:
I don't think this is possible because the client is not technically a part of the `arroyo-openapi` crate. Since it's generated by `arroyo-openapi/build.rs` it doesn't actually exist until build time.

Reply:
This is similar to how we have to use the `include!` macro for the generated api sql queries, I think it's just a consequence of the mono repo setup.

Reply:
Good idea, done.